src/site/content/en/identity/passkey-registration/index.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: Create a passkey for passwordless logins
+subhead: Passkeys make  user accounts safer, simpler, easier to use.
+authors:
+  - agektmr
+date: 2022-10-12
+description: |
+  Passkeys make a website's user accounts safer, simpler, easier to use and passwordless. This article discusses how to allow users to create passkeys for your website.
+tags:
+  - identity
+  - security
+---
+
+Using [passkeys](https://developers.google.com/identity/passkeys) instead of
+passwords is a great way for websites to make their user accounts safer,
+simpler, easier to use. With a passkey, a user can sign in to a
+website or an app just by using their fingerprint, face or device PIN.
+
+To enable passwordless sign in, a passkey first needs to be created and its public key stored on your server.
+
+When users sign in with a password, use the opportunity to ask them to create a passkey.
+
+Another good option is to have dedicated settings page where users can manage their passkeys and create new ones.
+
+## Prerequisites
+
+To create a passkey on Chrome for Android, you need the
+[Google Play services beta](https://developers.google.com/android/guides/beta-program).
+Otherwise the call to `navigator.credentials.create()` below will fail.
+
+## Create a new passkey
+
+### Feature detection
+
+Before prompting the user to create a new passkey, check if:
+
+-   The browser supports WebAuthn.
```

Comment:
you link to WebAuthn below, but this seems to be the first mention, so might be better to move that link here.